Dnn.Modules.Vendors.csproj.user

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can you please delete this file and add .csproj.user to the .gitingore file

Reply:
I got this resolved.